zammad/values.yaml

```
@@ -292,6 +292,8 @@ zammadConfig:
             - ALL
         readOnlyRootFilesystem: true
         privileged: false
+      # customCode: |
+      #   bundle exec rails runner '...'
```

i like an empty setting more. but no blocker...

```suggestion
      customInit: ""
        #  bundle exec rails runner '...'
```